content/francais/python-turtle/activity-4.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Activité 4 - Simplifier le code avec des fonctions"
+date: 2024-09-17T13:24:17-07:00
+weight: 6
+draft: false
+---
+
+Nous avons dessiné un hexagone. Mais ce n'est qu'un des nombreux hexagones qu'Alex doit construire pour sa ruche. Encore une fois, il semble que nous devrions répéter le code plusieurs fois pour dessiner plusieurs hexagones. Heureusement, nous n'en avons pas besoin ! Les fonctions à la rescousse !
```

Comment:
Change translation for accuracy.
"nous n'en avons pas besoin" to "nous n’avons pas à le faire"

Reply:
done